dockerfiles/piraeus-server/Dockerfile

```diff
@@ -74,5 +74,10 @@ ARG TARGETARCH
 
 RUN wget https://github.com/LINBIT/k8s-await-election/releases/download/${K8S_AWAIT_ELECTION_VERSION}/k8s-await-election-${K8S_AWAIT_ELECTION_VERSION}-linux-${TARGETARCH}.tar.gz -O - | tar -xvz -C /usr/bin/
 
+ARG LOSETUP_CONTAINER_VERSION=v1.0.1
+RUN wget "https://github.com/LINBIT/losetup-container/releases/download/${LOSETUP_CONTAINER_VERSION}/losetup-container-$(uname -m)-unknown-linux-gnu.tar.gz" -O - | tar -xvz -C /usr/local/sbin && \
+	 ln -snf losetup-container /usr/local/sbin/losetup
+ENV LOSETUP_CONTAINER_ORIGINAL_LOSETUP=/sbin/losetup
```

Comment:
I'm 100% sure we forget to change it to `/usr/sbin/losetup` when current 'testing' becomes the next stable. Is there a way to set `ENV` from the actual environment or from `$(which losetup)` (obviously before we add the new one). Or should we write the `$(which losetup)` to a file and source that later/run the satellite with `env`?

Reply:
Excellent point. Sadly it doesn't look like you can dynamically set ENV. Replaced the symlink with another script :shrug: